files/databases-list.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+res = {}
+
+for dbname in odoo.service.db.list_dbs(True):
+    res[dbname] = {}
+    registry = odoo.registry(dbname)
+    with registry.cursor() as cr:
+        cr.execute("SELECT key, value FROM ir_config_parameter WHERE key IN ('database.create_date', 'database.enterprise_code', 'database.expiration_date', 'database.expiration_reason', 'web.base.url')")
```

Comment:
Make the output order deterministic:
```suggestion
        cr.execute("SELECT key, value FROM ir_config_parameter WHERE key IN ('database.create_date', 'database.enterprise_code', 'database.expiration_date', 'database.expiration_reason', 'web.base.url') ORDER BY key")
```

Reply:
Fixed in 8687534f7f8ebf7ae12f414a7b9291a2330ba091